### DISCUSSION TITLE
I want activity only related with my repositories or watched repositories

### DISCUSSION THREAD

**Discussion by eusonlito** [community/community#66187] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I want a useful Github Home. I want to see only activity related with my repositories (stars, issues, PR, ...), my profile (follows) or related with repositories that I watch. I don't want suggestions, releases, or any other flood activity.

Please, consider to create a useful home to developers and stop to try to do another "great social network".

**Comment by kikearcos9999** [comment]
Hola, puedo hacer seguimiento a un dispositivo celular activado el gps (aplicación laravel gps tracking)
